config/locales/en/course/assessment/assessments.yml

```diff
@@ -75,7 +75,7 @@ en:
           failure: 'Could not delete assessment: %{error}'
         authenticate:
           header: 'Input Password'
-          message: 'The assessment is locked, please input the password.'
```

Comment:
Could we preserve the message and add another message instead saying that the assessment is not open yet?

Reply:
Done!